kong/api/routes/cluster.lua

```diff
@@ -90,6 +158,10 @@ return {
       -- Trigger event in the node
       singletons.events:publish(message_t.type, message_t)
 
+      -- Increment counter
+      increment(cache.event_key(), 1)
+      increment(cache.event_key(message_t.type or OTHER), 1)
```

Comment:
Those benefit little, if none, from such an abstraction (in the `increment()` function). If anything they just make future contributors have to go all the way up the file to read `increment` which should only be 3 LOC...
Use:
```lua
local cache_dict = ngx.shared.cache
--- ... 
local ok, err = cache_dict:incr("my_key", 1, 0)
if not ok then 
  ngx.log(ngx.ERR, "could not increment [my_key]: ", err)
end

ok, err = cache_dict:incr("my_other_key", 1, 0)
if not ok then 
  ngx.log(ngx.ERR, "could not increment [my_other_key]: ", err)
end
```

Simple and explicit.